algebras/algebra/src/main/scala/endpoints4s/algebra/ChunkedEntities.scala

```diff
@@ -77,24 +93,56 @@ trait ChunkedEntities extends EndpointsWithCustomErrors {
   *
   * @group algebras
   */
-trait ChunkedJsonEntities extends ChunkedEntities with JsonCodecs {
+trait ChunkedJsonEntities extends ChunkedJsonRequestEntities with ChunkedJsonResponseEntities
+
+trait ChunkedJsonRequestEntities
+    extends ChunkedRequestEntities
+    with JsonCodecs
+    with RequestChunkCodec {
 
   /** A request entity carrying chunks of JSON values
     *
     * @tparam A Type of values serialized into JSON
     * @group operations
     */
-  def jsonChunksRequest[A](implicit
+  def jsonChunksRequest[A](chunkCodec: RequestChunkCodec)(implicit
       codec: JsonCodec[A]
   ): RequestEntity[Chunks[A]]
+}
+
+trait ChunkedJsonResponseEntities
+    extends ChunkedResponseEntities
+    with JsonCodecs
+    with ResponseChunkCodec {
 
   /** A response entity carrying chunks of JSON values
     *
     * @tparam A Type of values serialized into JSON
     * @group operations
     */
-  def jsonChunksResponse[A](implicit
+  def jsonChunksResponse[A](chunkCodec: ResponseChunkCodec)(implicit
```

Comment:
Unfortunately we can’t just break the API. We should keep the previous method (maybe deprecate it) and introduce an overload, instead of changing it.